packages/gestalt/src/Touchable.css

```diff
@@ -1,5 +1,7 @@
 .touchable {
   composes: accessibilityOutline from "./Focus.css";
+  transition: transform .2s ease-out;
+  outline: 0;
```

Comment:
Adding `outline` here for consistency with what we do on mweb today. Not sure if it is okay. Open to suggestions.

Reply:
@jackhsu978 take a look at https://github.com/pinterest/gestalt/blob/bcc420b7e3e2086b165757b7e7ab35f0517fe9a7/packages/gestalt/src/Focus.css#L3 - we already set `outline: 0` when we compose from `./Focus.css`